shanoir-ng-studies/src/main/java/org/shanoir/ng/subject/controler/SubjectApi.java

```diff
@@ -68,7 +70,12 @@ ResponseEntity<Void> deleteSubject(
 	@GetMapping(value = "", produces = { "application/json" })
 	@PreAuthorize("hasAnyRole('ADMIN', 'EXPERT', 'USER')")
 	@PostAuthorize("hasRole('ADMIN') or @studySecurityService.filterSubjectDTOsHasRightInOneStudy(returnObject.getBody(), 'CAN_SEE_ALL')")
-	ResponseEntity<List<SubjectDTO>> findSubjects();
+	ResponseEntity<List<SubjectDTO>> findSubjects(
```

Comment:
same : Is this necessary to have two booleans when a subject is either clinical or preclinical ?